tests/minigraph/test_masked_services.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import time
+
+from tests.common.helpers.assertions import pytest_assert
+
+def is_service_up(duthost, service):
+    output = duthost.shell("sudo systemctl show -p LoadState --value {}.service".format(service))
+    if output['stdout'] != "loaded":
+        return False
+    return True
+
+def change_service_state(duthost, service, enable):
+    if not enable:
+        duthost.shell("sudo systemctl stop {}.service".format(service))
+        duthost.shell("sudo systemctl disable {}.service".format(service))
+        duthost.shell("sudo systemctl mask {}.service".format(service))
+    else:
+        duthost.shell("sudo systemctl unmask {}.service".format(service))
+        duthost.shell("sudo systemctl enable {}.service".format(service))
+        duthost.shell("sudo systemctl start {}.service".format(service))
+
+def test_masked_services(duthosts, rand_one_dut_hostname):
+    """
+    @summary: This test case will mask a running service, then test load_minigraph and check its success
+    """
+
+    duthost = duthosts[rand_one_dut_hostname]
+
+    logging.info("Bringing down a running service")
+
+    test_service = ""
+    services = [
+        "telemetry",
+        "restapi",
+        "bgp",
+        "lldp",
+        "pmon",
+        "swss",
+        "nat"
+    ]
+   
+    for service in services:
+        service_status = duthost.critical_process_status(service)
+        if service_status:
+            test_service = service
+            break
+
+    if not test_service:
+        logging.error("No valid service to run test on.")
+        return
```

Comment:
>return [](http://example.com/codeflow?start=8&length=6)

Does `return` here mean testcase passing? Actually nothing is tested and we should treat it as failure.